packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -404,6 +404,9 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
       safari10: true,
       output: {
         ecma: terserEcma,
+        // For differential loading, this is handled in the bundle processing.
+        // The experimental rollup support breaks without this check.
+        ascii_only: !differentialLoadingMode,
```

Comment:
@clydin I am not following how the experimental rollup support is effected by the `differentialLoadingMode` flag

Reply:
The experimental rollup E2E test fails if this is always set to true.  Everything else passes without issue and it technically should work if just true.  But the functionality is covered either way.  I just wanted to mention it here in case someone decides to simplify this later.